python/pyspark/sql/functions.py

```diff
@@ -44,6 +45,14 @@ def _(col):
     return _
 
 
+def _wrap_deprecated_function(func, message):
+    """ Wrap the deprecated function to print out deprecation warnings"""
+    def _(col):
+        warnings.warn(message, DeprecationWarning)
+        return func(col)
```

Comment:
Here, I intendedly avoided `*args` and `**kwargs` to keep the argument signature printed in pydoc, `help(...)`.